src/web/browser/startup.ts

```diff
@@ -14,8 +14,12 @@ const measure = (name: string, task: () => Promise<void>): void => {
         // in Chrome 59 which is used by Cypress in headless mode so if you do it will
         // break the Cypress tests on CI
         // See: https://github.com/cypress-io/cypress/issues/2651#issuecomment-432698837
-        .then(end)
-        .catch(end);
+        .then(() => {
+            end()
+        })
+        .catch(()=> {
+            end()
+        });
```

Comment:
Isn't think equivalent to what it was originally?

Reply:
We were seeing some Typescript errors and @tjmw came up with this solution - it solved the problem, but it slightly went over my head - we're going to run over it once he's back.

Reply:
Yeah it's because we've changed `end` to return the time taken, but didn't want the knock on effect of changing the interface/return value here. So we're maintaining `measure`'s return type of `Promise<void>`.